cmd/sriov/main.go

```diff
@@ -207,10 +226,20 @@ func cmdDel(args *skel.CmdArgs) error {
 		}
 	}
 
+	vfPciAddress, err := utils.GetPciAddress(netConf.Master, netConf.VFID)
+	if err != nil {
+		return fmt.Errorf("error getting vf pci address for PF %s and VF id %d: %v", netConf.Master, netConf.VFID, err)
+	}
+
 	if err := sm.ResetVFConfig(netConf); err != nil {
 		return fmt.Errorf("cmdDel() error reseting VF: %q", err)
 	}
 
+	// Mark the pci address as released
```

Comment:
There is a risk we may never reach here in case cmdDel errors early.

also i assume runtime will __always__ call CNI delete command, is that the case ?

Reply:
@adrianchiris please take a look now I add a method to be sure we don't block the PCI address for ever